modules/tinymce/src/core/main/ts/fmt/MatchFormat.ts

```diff
@@ -67,65 +67,65 @@ const matchName = (dom: DOMUtils, node: Node, format) => {
   }
 };
 
-const matchItems = (dom: DOMUtils, node: Node, format, itemName: string, similar: boolean, vars: FormatVars) => {
-  let key, value;
+const matchItems = (dom: DOMUtils, node: Node, format: Format, itemName: string, similar: boolean, vars: FormatVars): boolean => {
   const items = format[itemName];
-  let i;
 
   // Custom match
   if (format.onmatch) {
-    return format.onmatch(node, format, itemName);
+    const onMatch: (node: Node, format: Format, name: string) => boolean = format.onmatch;
+    return onMatch(node, format, itemName);
```

Comment:
Why was this changed? This is very ugly and doesn't add anything, so I'd like to see this reverted please.
```suggestion
    return format.onmatch(node, format, itemName);
```

Reply:
I see you said why after I'd added this in my review. That seems like a bad reason to be doing this and instead I'd prefer the `format: Format` type be removed above instead until we can resolve what's going on.

Reply:
Have changed this to a different cast that's less ugly but still more correct than just leaving format to be an implicit any.